bigquery/table.go

```diff
@@ -160,6 +160,10 @@ type TableMetadata struct {
 	// where 12345 is parent id. The value is the friendly short name of the
 	// tag value, e.g. "production".
 	ResourceTags map[string]string
+
+	// MaxStaleness of data that could be returned when materialized
```

Comment:
I think the description may need more detail.  My understanding is table max_staleness can be relevant to several things (views, federation queries)?